package.json

```diff
@@ -24,11 +30,11 @@
     "recharts": "^1.0.0-beta.10"
   },
   "scripts": {
-    "start": "react-scripts start",
+    "start": "echo 'This script has changed. Please use `yarn web` or `yarn server` instead.'",
+    "web": "react-scripts start",
     "build": "react-scripts build",
-    "test": "react-scripts test --env=jsdom",
-    "predeploy": "npm run build",
-    "deploy": "gh-pages -d build",
-    "eject": "react-scripts eject"
+    "server": "NODE_ENV=development nodemon --watch server --exec 'node server/index'",
```

Comment:
Windows was stupid and I had to replace  
"NODE_ENV=development nodemon --watch server --exec 'node server/index'"

with
"server": "SET NODE_ENV=development & nodemon server/index",
to get it working. couldn't get the --watch server --exec working. Tried it on a linux machine and it worked just fine though

Reply:
Went back and did it in a Linux VM and it worked fine. MongoDB has a different syntax then the one our backend tries to use to boot, and because of that its not correctly starting up on Windows. I attempted some fixes but nothing really worked.